Islandora-Devops/isle-dc
# Add basic https entrypoint with Traefik self-signed default certificate

For development purposes, add a basic https entrypoint with Traefik self-signed default certificate
* Drupal
  * add plumbing for secure http endpoint
  * add config to redirect http to https
  * set isDevelopment property to `true` thus disabling the redirect for MVP1

To test:
* clone repo
* ` docker-compose -f docker-compose.mvp1.yml pull`
* ` docker-compose -f docker-compose.mvp1.yml up -d`
* on linux, add `idcp.localdomain` to `/etc/hosts` as an alias for localhost
* ` lynx http://idcp.localdomain:80/`
* if this works, one should see a Drupal 8.8.1 Installation tasks
* ` lynx https://idcp.localdomain`
  * accept self-signed cert
* if this works, one should see a Drupal 8.8.1 Installation tasks

Where `idcp.localdomain` is the `.env` DOMAIN property
